README.md

```diff
@@ -134,7 +134,7 @@ to report problems, discuss roadmap items, or make feature requests.
 If you've discovered an issue that may have security implications to
 users or developers of this software, please do not report it using
 GitHub issues, but instead follow
-[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/master/SECURITY-POLICY.md).
+[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/main/SECURITY-POLICY.md).
```

Comment:
Would you mind updating the link to reflect the file rename too?  It looks like the file got renamed to `SECURITY.md`.

```suggestion
[Firecracker's security reporting guidelines](https://github.com/firecracker-microvm/firecracker/blob/main/SECURITY.md).
```

(the above link doesn't work; I assume that the branch rename has not completed yet?)

Reply:
I think so. Let's rename the branch on Firecracker first. So that we can merge this change.

Reply:
@samuelkarp, you are right, the branch rename has not been completed yet, but will soon. I updated the PR to include the file name change you pointed out.